include/engine/geospatial_query.hpp

```diff
@@ -545,6 +578,32 @@ template <typename RTreeT, typename DataFacadeT> class GeospatialQuery
         return datafacade.GetComponentID(data.forward_segment_id.id).is_tiny;
     }
 
+    std::pair<bool, bool> approach_usage(const util::Coordinate &input_coordinate,
```

Comment:
Rename to `ApproachUsage` for camel-case. A better name could be `CheckApproach`.